app/src/main/java/io/mochahub/powermeter/MainActivity.kt

```diff
@@ -13,14 +16,18 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
         AppCenter.start(application, BuildConfig.HOCKEY_APP_SECRET,
             Push::class.java,  Analytics::class.java, Crashes::class.java)
         Analytics.trackEvent("Hello World")
 
-        supportFragmentManager
-            .beginTransaction()
-            .add(R.id.fragmentLayout, MainFragment.newInstance(), "mainFragment")
-            .commit()
+        setContentView(R.layout.activity_main)
+        val navController = Navigation.findNavController(this, R.id.nav_host_fragment)
+        setupBottomNavMenu(navController)
+    }
+
+    private fun setupBottomNavMenu(navController: NavController) {
+        bottom_navigation?.let {
+            NavigationUI.setupWithNavController(it, navController)
+        }

```

Comment:
I think this can all be replaced with `NavigationUI.setupWithNavController(bottom_navigation, navController)`

1. I don't see a need for an extracted `setupBottomNavMenu` function if it isn't being used elsewhere
2. I don't think it needs the `?.` if its only being called in `onCreate` since by then the view is created and it shouldn't be null